content/en/tracing/trace_collection/compatibility/dotnet-core.md

```diff
@@ -49,12 +49,60 @@ The .NET Tracer supports automatic instrumentation on the following architecture
 
 | Processor architectures                   | Support level         | Package version                        |
 | ------------------------------------------|-----------------------|----------------------------------------|
-| Windows x86 (`win-x86`)                   | [GA](#support-ga)     | latest                                 |
 | Windows x64 (`win-x64`)                   | [GA](#support-ga)     | latest                                 |
+| Windows x86 (`win-x86`)                   | [EOL](#support-eol)   | < 3.0.0 (e.g. 2.48.0)                  |
 | Linux x64 (`linux-x64`)                   | [GA](#support-ga)     | latest                                 |
 | Alpine Linux x64 (`linux-musl-x64`)       | [GA](#support-ga)     | latest                                 |
 | Linux ARM64 (`linux-arm64`)               | [GA](#support-ga)     | .NET 5+ only, added in version 1.27.0  |
 
+Note that running x86 applications on Windows x64 is supported.
+
+## Supported operating systems
+
+The .NET Tracer supports automatic instrumentation on Windows and Linux operating systems. It supports macOS for CI Test Visibility only.
+
+### Windows
+
+| Operating System             | Version     | Support level         | Package version                        |
+| -----------------------------|-------------|-----------------------|----------------------------------------|
+| Windows Server (x64)         | 2012+       | [GA](#support-ga)     | latest                                 |
+| Windows Client (x64)         | 8.1+        | [GA](#support-ga)     | latest                                 |
+| Nano Server (x64)            | < 2012      | [EOL](#support-eol)   | < 3.0.0 (e.g. 2.48.0)                  |
+| Windows Server (x64)         | < 2012      | [EOL](#support-eol)   | < 3.0.0 (e.g. 2.48.0)                  |
+| Windows Server (x86)         |             | [EOL](#support-eol)   | < 3.0.0 (e.g. 2.48.0)                  |
```

Comment:
So I left this blank, because we consider support for _all_ x86 versions EOL and they should use < 3.0.0 🤔

Reply:
Ah my mistake. Maybe it would be clearer if we put `All versions` in that section instead of leaving it blank, but I don't have strong opinions so I'll go away now 😄